tests/kamereon/test_kamereon_vehicle_data.py

```diff
@@ -304,6 +304,36 @@ def test_location_v1() -> None:
     assert vehicle_data.gpsLongitude == 11.1234567
     assert vehicle_data.lastUpdateTime == "2020-02-18T16:58:38Z"
 
+def test_lock-status_v1() -> None:
```

Comment:
You are not allowed dashes in the method name.
Also could you move the method down after location_v2?
```suggestion
def test_lock_status_v1() -> None:
```

Reply:
Should be fixed now